api/embedded_profile.asciidoc

```diff
@@ -305,3 +305,30 @@ Device Queries>> table is {CL_TRUE}, the values assigned to
 must be greater than or equal to the minimum values specified in the
 <<embedded-device-queries-table, OpenCL Embedded Device Queries>> table.
 
+For 1D, 1D image from buffer, 2D, 3D image objects, 1D and 2D image array
+objects, when supported, the mandated minimum list of image formats that can
+be read from and written to by different kernel instances when correctly ordered
+by event dependencies and that must be supported by all devices that support
+images is described in the <<embedded-required-image-read-or-write-formats-table>> table.
+
+[[embedded-required-image-read-or-write-formats-table]]
+.Minimum list of required image formats (embedded profile): kernel read or write
+[width="100%",cols="<34%,<33%,<33%",options="header"]
+|====
+| num_channels | channel_order | channel_data_type
+| 4
+  | {CL_RGBA}
+      | {CL_UNORM_INT8} +
+        {CL_UNORM_INT16} +
+        {CL_SIGNED_INT8} +
+        {CL_SIGNED_INT16} +
+        {CL_SIGNED_INT32} +
+        {CL_UNSIGNED_INT8} +
+        {CL_UNSIGNED_INT16} +
+        {CL_UNSIGNED_INT32} +
+        {CL_HALF_FLOAT}^1^ +
```

Comment:
Is this footnote required?

I don't see a similar footnote in the OpenCL 1.2 spec or for the full profile, and it feels strange to me to tie the `CL_HALF_FLOAT` image storage format to the `cl_khr_fp16` extension that is otherwise all about computation on the fp16 ("`half`") type.

I'd prefer to either remove the footnote or remove the `CL_HALF_FLOAT` image format from the minimum list of required formats unconditionally.

Reply:
It's actually not required. I'll remove it.

Reply:
BTW, removing `CL_HALF_FLOAT` from the list of formats wouldn't have been an option since the format was required by OpenCL 1.2.

Reply:
Thanks!  Latest changes LGTM.